cas-server-support-oidc/src/main/java/org/apereo/cas/config/OidcServerDiscoveryProperties.java

```diff
@@ -6,22 +6,24 @@
 import java.util.List;
 
 /**
- * This is {@link ServerDiscoveryProperties}.
+ * This is {@link OidcServerDiscoveryProperties}.
  *
  * @author Misagh Moayyed
  * @since 5.0.0
  */
-public class ServerDiscoveryProperties {
-    private String issuer;
+public class OidcServerDiscoveryProperties {
+    
     private List<String> supportedScopes;
     private List<String> supportedResponseTypes;
     private List<String> supportedSubjectTypes;
     private List<String> supportedClaimTypes;
     private List<String> supportedClaims;
     private List<String> supportedGrantTypes;
-    private String serverPrefix;
 
-    public ServerDiscoveryProperties(final String serverPrefix, final String issuer) {
+    private final String issuer;
+    private final String serverPrefix;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['VARIABLE_DEF' should be separated from previous statement.](https://www.codacy.com/app/mmoayyed/cas/file/2612810342/issues/source?bid=3256115&fileBranchId=3310321#l24)